### PR TITLE
add guard to offload to gpu

### PR DESCRIPTION
**Pull request by xyyimian**

### CODE REVIEW

**Comment by xyyimian** [comment]
test and it worked locally to solve my mac intel chip issue

File: nexa/gguf/llama/llama.py [hunk 1]
```diff
@@ -211,6 +212,7 @@ def __init__(
         self.model_path = model_path
 
         # Model Params
+        n_gpu_layers = n_gpu_layers if is_gpu_available() else 0
```
Comment (AgainstEntropy) [inline nexa/gguf/llama/llama.py:215]: maybe it's more proper to put this line in `nexa_inference_text.py`?

**Comment by zhiyuan8** [comment]
@AgainstEntropy 
please modify this to nexa_infernece_text and vlm scripts and I will merge

**Comment by xyyimian** [comment]
@AgainstEntropy either we made change for text vlm and nexa_service.py, or we change in low level llama.py.

**Pull request merged**